beacon_node/src/config.rs

```diff
@@ -265,34 +263,6 @@ pub fn get_config<E: EthSpec>(
         client_config.http_metrics.allocator_metrics_enabled = false;
     }
 
-    /*
-     * Eth1
-     */
-
-    if cli_args.get_flag("dummy-eth1") {
-        warn!("The --dummy-eth1 flag is deprecated");
-    }
-
-    if cli_args.get_flag("eth1") {
-        warn!("The --eth1 flag is deprecated");
-    }
-
-    if let Some(val) = cli_args.get_one::<String>("eth1-blocks-per-log-query") {
-        client_config.eth1.blocks_per_log_query = val
-            .parse()
-            .map_err(|_| "eth1-blocks-per-log-query is not a valid integer".to_string())?;
-    }
-
-    if cli_args.get_flag("eth1-purge-cache") {
-        client_config.eth1.purge_cache = true;
-    }
-
-    if let Some(follow_distance) =
-        clap_utils::parse_optional(cli_args, "eth1-cache-follow-distance")?
-    {
-        client_config.eth1.cache_follow_distance = Some(follow_distance);
-    }
```

Comment:
We should probably keep these 3 flags and log warnings here, seeing as we had not previously deprecated them. Usually we try to avoid fully removing flags unless they've been deprecated first.